SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -210,29 +210,32 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         routeAudioToSpeaker()
     }
     
-    func callDidDisconnect(_ call: TVOCall) {
-        NSLog("callDidDisconnect:")
-
-        performEndCallAction(uuid: call.uuid)
-
-        self.call = nil
-        self.callKitCompletionCallback = nil
+    func call(_ call: TVOCall, didFailToConnectWithError error: Error) {
+        NSLog("Call failed to connect: \(error.localizedDescription)")
         
-        self.placeCallButton.setTitle("Place Outgoing Call", for: .normal)
+        performEndCallAction(uuid: call.uuid)
+        callDisconnected()
+    }
```

Comment:
Shouldn't this method implement the same logic as previously in:

`func call(_ call: TVOCall?, didFailWithError error: Error`

Namely, ending the call action and firing `self.callKitCompletionCallback` with a failure? The former is done, but not the latter.